migrations/20170619153242_increase-outcome-column-width.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex => Promise.all([
+  knex('assessments').update('viper', 0),
```

Comment:
this shouldn't still be here - should only be the schema call, and no `Promise.all`. Same for the down migration